src/Dumper.php

```diff
@@ -56,7 +56,7 @@ public function asJson(int $depth = 50, bool $format = false): string
     public function asJsonObjectsMap(int $depth = 50, bool $prettyPrint = false): string
     {
         $this->buildObjectsCache($this->variable, $depth);
-        return $this->asJsonInternal($this->objects, $prettyPrint, $depth, 1, true);
+        return $this->asJsonInternal($this->objects, $prettyPrint, 2, 1, true);
```

Comment:
Could be that 2 is more than enough and 1 would do as well.

Reply:
> 1 would do as well

2 is min, because property of the root object, that is object, must be converted to the expected string
https://github.com/yiisoft/yii-debug/blob/90bb1960841cf3b054433d7c54210fd203d3fe8f/src/Dumper.php#L150

but this works only if dumped variable object or function, if it's an array, than it must be dumped deeper (`or no?`).